db/db.js

```diff
@@ -63,7 +63,7 @@ async function rollbackTransaction(client) {
 
 async function runQuery(functionName, queryString, queryParams, clientParam) {
   let client = clientParam
-  const transactionMode = typeof client !== 'undefined'
+  const transactionMode = client !== undefined
```

Comment:
For consistency with how we do this comparison elsewhere in the codebase. A good discussion here: https://stackoverflow.com/questions/3390396/how-can-i-check-for-undefined-in-javascript#answer-3390635